swap/peer.go

```diff
@@ -65,8 +65,6 @@ func (sp *Peer) handleMsg(ctx context.Context, msg interface{}) error {
 
 // handleEmitChequeMsg should be handled by the creditor when it receives
 // a cheque from a debitor
-// TODO: validate the contract address in the cheque to match the address given at handshake
-// TODO: this should not be blocking
```

Comment:
removed. The function calling this function (swap/peer.go handleMsg) is concurrent (it probably was not like this previously)